webapp/src/theme.ts

```diff
@@ -183,7 +183,7 @@ export function setMattermostTheme(theme: any): Theme {
     document.documentElement.style.setProperty('--sidebar-bg-rgb', color(theme.sidebarBg).rgb().array().join(', '))
     document.documentElement.style.setProperty('--sidebar-text-rgb', color(theme.sidebarText).rgb().array().join(', '))
     document.documentElement.style.setProperty('--link-color-rgb', theme.linkColor)
-    document.documentElement.style.setProperty('--sidebar-text-active-border', color(theme.sidebarTextActiveBorder).rgb().array().join(', '))
+    document.documentElement.style.setProperty('--sidebar-text-active-border-rgb', color(theme.sidebarTextActiveBorder).rgb().array().join(', '))
```

Comment:
just a question: do you still need to call `rgb()` since you already get the rgb channel values in the theme variable, or is this for safety purposes (when hex values come in instead)?

Reply:
Yes, exactly is for safety purposes. Shouldn't be hex, but this way can be in any supported format.